src/state/MuteAllAudioModel.test.ts

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2025 New Vector Ltd.
+
+SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-Element-Commercial
+Please see LICENSE in the repository root for full details.
+*/
+
+import { test, vi } from "vitest";
+import { expect } from "vitest";
+
+import { setOutputEnabled$ } from "../controls";
+import { muteAllAudio as muteAllAudioSetting } from "../settings/settings";
+import { muteAllAudio$ } from "./MuteAllAudioModel";
+
+test("muteAllAudio$", () => {
```

Comment:
Not a blocker, but this would be more idiomatic as a marble test

Reply:
I agree. Since it s not a blocker I prefer postponing this to a later PR.